app/views/recipes/show.html.erb

```diff
@@ -4,6 +4,16 @@
   <ul>
     <li><%= @recipe.ingredients %></li>
     <li><%= @recipe.instructions %></li>
+    <% if @recipe.pictures.attached? %>
+      <li>
+        <p>
+        <strong>Pictures:</strong>
+        <br>
```

Comment:
What would you think about controlling this break with css?

Alternatively, I think `<strong>Pictures:</strong>` and the `.each` loop could be placed inside their own `<div>` tags, which would likely display them on new lines. 

:memo: My inspiration comes from ideas like [this SO post](https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br). Id like to note that its from 2009, but I am assuming this is still valid in 2019.

Reply:
I'd much rather see a `dl`/`dt`/`dd` than a `div`, personally. This is inline with [the upcoming accessibility guide](https://github.com/thoughtbot/guides/pull/579): Ensure semantic markup is used to describe content.

Reply:
Thanks for bringing that to light - and I also agree :+1:

Reply:
Thanks for the insight. I was unfamiliar with `dl/dt/dd` previously, but this makes sense to have this file formatted as such.